_layouts/recipe.html

```diff
@@ -46,10 +46,21 @@
                 <dt>Makes</dt>
                 <dd>{{ page.makes }}</dd>
                 <dt>Category</dt>
-                <dd>{{ page.category }}</dd>
-              </dl>
+                <dd><a href = "{{ page.category | downcase | prepend: 'search.html?category=' | prepend: site.baseurl }}">{{ page.category }}</a></dd>
+                {% capture difference %} {{ page.tags | size | minus:1 }} {% endcapture %}
+                {% unless difference contains '-' %}
+                <dt>Tags</dt>
+                <dd style="overflow-x: scroll; white-space: nowrap;">
+                  {% assign sorted_tags = (page.tags | sort) %}
+                  {% for tag in sorted_tags %}
+                <a href="{{ tag | downcase | prepend: 'search.html?tags=' | prepend: site.baseurl }}">{{ tag }}</a>
+                    {% if forloop.last == false %}, {% endif %}
```

Comment:
`{% if not forloop.last %}`, or `{% unless forloop.last %}` would be clearer here